static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-control.js

```diff
@@ -15,21 +15,23 @@ const buildImage = (url: string): string =>
 export const acquisitionsEpicControlTemplate = ({
     copy: { heading = '', paragraphs, highlightedText, footer },
     componentName,
+    epicClassNames = [],
     buttonTemplate,
     wrapperClass = '',
     showTicker = false,
     backgroundImageUrl,
 }: {
     copy: AcquisitionsEpicTemplateCopy,
     componentName: string,
+    epicClassNames: string[],
     buttonTemplate?: string,
     wrapperClass?: string,
     showTicker: boolean,
     backgroundImageUrl?: string,
 }) => {
-    const extraClasses = backgroundImageUrl
-        ? 'contributions__epic--with-image'
-        : '';
+    const extraClasses = epicClassNames
```

Comment:
To keep the empty string out of the array could we do something like:
```
const extraClasses = backgroundImageUrl ? epicClassNames.push('contributions__epic--with-image') : epicClassNames
```

I guess it doesn't matter so much because we are turning it into a string